File: Dot Net Api's/EduNova/EduNova/Models/Student.cs
using System;
using System.Collections.Generic;

namespace EduNova.Models;

public partial class Student
{
    public int StudentId { get; set; }

    public int? UId { get; set; }

    public string FirstName { get; set; } = null!;

    public string LastName { get; set; } = null!;

    public string Email { get; set; } = null!;

    public string? PhoneNo { get; set; }

    public string? Qualification { get; set; }

    public bool? Enable { get; set; }

    public virtual ICollection<StudentExam> StudentExams { get; set; } = new List<StudentExam>();

    public virtual ICollection<StudentResult> StudentResults { get; set; } = new List<StudentResult>();

    public virtual User? UIdNavigation { get; set; }
}

File: edunaova_spring/demo/src/main/java/com/example/demo/entities/Student.java
package com.example.demo.entities;

import jakarta.persistence.*;

@Entity
public class Student {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int studentId;

    @Column(nullable = false)
    private String firstName;

    @Column(nullable = false)
    private String lastName;

    @Column(nullable = false)
    private String email;

    @Column
    private String phoneNo;

    @Column
    private String qualification;

    @Column
    private boolean enable;

    @OneToOne
    @JoinColumn(name = "uId")
    private User user;

	public Student() {
		
	}

	public Student(int studentId, String firstName, String lastName, String email, String phoneNo, String qualification,
			boolean enable, User user) {
		super();
		this.studentId = studentId;
		this.firstName = firstName;
		this.lastName = lastName;
		this.email = email;
		this.phoneNo = phoneNo;
		this.qualification = qualification;
		this.enable = enable;
		this.user = user;
	}

	public int getStudentId() {
		return studentId;
	}

	public void setStudentId(int studentId) {
		this.studentId = studentId;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getPhoneNo() {
		return phoneNo;
	}

	public void setPhoneNo(String phoneNo) {
		this.phoneNo = phoneNo;
	}

	public String getQualification() {
		return qualification;
	}

	public void setQualification(String qualification) {
		this.qualification = qualification;
	}

	public boolean isEnable() {
		return enable;
	}

	public void setEnable(boolean enable) {
		this.enable = enable;
	}

	public User getUser() {
		return user;
	}

	public void setUser(User user) {
		this.user = user;
	}

	
   
}
